### PR TITLE
feat: add stripe test key fields to settings model

### LINKED ISSUE #6321: Add stripe test key fields in DB
Currently, there are only stripe client ID, secret key and publishable key in settings model, there should be test keys fields and they should be used when Debug is true

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #6321

#### Short description of what this resolves:
Currently, there are only stripe client ID, secret key and publishable key in settings model, there should be test keys fields and they should be used when Debug is true

#### Changes proposed in this pull request:
- adds stripe test keys to settings model
- need to add a check to use these keys is debug is true

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: migrations/versions/rev-2019-08-06-15:29:45-b3606cb4dace_.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'b3606cb4dace'
+down_revision = '4925dd5fd720'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('settings', sa.Column('stripe_test_client_id', sa.String(), nullable=True))
```
Comment (CosmicCoder96) [inline migrations/versions/rev-2019-08-06-15:29:45-b3606cb4dace_.py]: client id is same for test and live keys.

File: app/api/schema/settings.py [hunk 2]
```diff
@@ -82,6 +82,8 @@ class Meta:
     # Stripe Keys
     stripe_client_id = fields.Str(allow_none=True)
     stripe_publishable_key = fields.Str(allow_none=True)
+    stripe_test_client_id = fields.Str(allow_none=True)
+    stripe_test_publishable_key = fields.Str(allow_none=True)
```
Comment (CosmicCoder96) [inline app/api/schema/settings.py]: test_secret_key is missing.

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 made the requested changes. Please have a look

File: app/api/helpers/payment.py [hunk 3]
```diff
@@ -40,7 +40,14 @@ def get_credentials(event=None):
...
+            if settings['app_environment'] == 'development' and settings['stripe_test_secret_key'] and \
+                settings['stripe_test_publishable_key'] and settings['stripe_test_secret_key'] != "" and \
+                    settings['stripe_test_publishable_key'] != "":
+                return {
+                    'SECRET_KEY': settings['stripe_test_secret_key'],
+                    'PUBLISHABLE_KEY': settings["stripe_test_publishable_key"]
+                }
+            elif settings['stripe_secret_key'] and settings["stripe_publishable_key"] and settings[
                 'stripe_secret_key'] != "" and \
                     settings["stripe_publishable_key"] != "":
```
Comment (iamareebjamal) [inline app/api/helpers/payment.py]: Redundant checks. Empty string is Falsy in python

Comment (shreyanshdwivedi) [reply app/api/helpers/payment.py]: Sorry for the redundant check. I fixed it at another place where it was occurring

**Comment by shreyanshdwivedi** [comment]
@iamareebjamal @mrsaicharan1 @uds5501 please review

**Pull request merged**